src/EnergyPlus/DataSurfaces.cc

```diff
@@ -677,6 +694,191 @@ namespace DataSurfaces {
+            }
+        }
+
+        return RefAirTemp;
+    }
+
+    Real64 SurfaceData::getInsideIR(const int t_SurfNum)
+    {
+        auto &window(SurfaceWindow(t_SurfNum));
+        const Real64 value = window.IRfromParentZone + QHTRadSysSurf(t_SurfNum) + QHWBaseboardSurf(t_SurfNum) + QSteamBaseboardSurf(t_SurfNum) +
+                             QElecBaseboardSurf(t_SurfNum);
+        return value;
+    }
+
+    Real64 SurfaceData::getOutsideAirTemperature(const int t_SurfNum) const
+    {
+        // SUBROUTINE INFORMATION:
+        //       AUTHOR         Simon Vidanovic
+        //       DATE WRITTEN   June 2016
+        //       MODIFIED       na
+        //       RE-ENGINEERED  na
+
+        // PURPOSE OF THIS SUBROUTINE:
+        // Routine calculates outside air temperature for given surface.
+        // Routine will return inside air temperature if it is interior surface. (refactoring from the code)
+        //
+        // NOTE: This routine has been copy/pasted in the past in several different modules with slight
+        //       modifications at some of those places. Exterior/interior surface air temperature is tied to surface.
+        Real64 temperature = 0;
+
+        if (ExtBoundCond > 0) // Interzone window
+        {
+            temperature = getInsideAirTemperature(t_SurfNum);
+        } else {
+            if (ExtWind) {
+                // Window is exposed to wind (and possibly rain)
+                if (IsRain) {
+                    // Raining: since wind exposed, outside window surface gets wet
+                    temperature = OutWetBulbTemp;
+                } else {
+                    // Dry
+                    temperature = OutDryBulbTemp;
+                }
+            } else {
+                // Window not exposed to wind
+                temperature = OutDryBulbTemp;
+            }
+        }
+
+        return temperature;
+    }
+
+    Real64 SurfaceData::getOutsideIR(const int t_SurfNum) const
+    {
+        // SUBROUTINE INFORMATION:
+        //       AUTHOR         Simon Vidanovic
+        //       DATE WRITTEN   July 2016
+        //       MODIFIED       na
+        //       RE-ENGINEERED  na
+
+        // PURPOSE OF THIS SUBROUTINE:
+        // Calculates outside infrared radiation
+        Real64 value = 0;
+        if (ExtBoundCond > 0) {
+            value = SurfaceWindow(ExtBoundCond).IRfromParentZone + QHTRadSysSurf(ExtBoundCond) + QHWBaseboardSurf(ExtBoundCond) +
+                    QSteamBaseboardSurf(ExtBoundCond) + QElecBaseboardSurf(ExtBoundCond);
+        } else {
+            Real64 tout = getOutsideAirTemperature(t_SurfNum) + KelvinConv;
+            value = sigma * pow_4(tout);
+            value = ViewFactorSkyIR * (AirSkyRadSplit(t_SurfNum) * sigma * pow_4(SkyTempKelvin) + (1.0 - AirSkyRadSplit(t_SurfNum)) * value) +
+                    ViewFactorGroundIR * value;
+        }
+        return value;
+    }
+
+    Real64 SurfaceData::getSWIncident(const int t_SurfNum)
```

Comment:
Should these little `get` functions be `inline`d?  Maybe the compiler is already inlining them.

Reply:
I think static is inline by default. Not sure.
BTW, I was extracting some common routines out. Like Short-Wave incident incoming to surface is calculated on so many places using same equation:

QRadSWOutIncident(t_SurfNum) + QS(Surface(t_SurfNum).Zone)

I was wondering maybe we can use clean E+ even more by identifying such functions.

Reply:
As long as they are inlined then I'm good with it.  I just don't want accessor functions to add any overhead.